libnmstate/validator.py

```diff
@@ -250,9 +250,20 @@ def _assert_vxlan_has_missing_attribute(state, *attributes):
 
 def _assert_vlan_filtering_trunk_tags(ports_state):
     for port_state in ports_state:
-        trunk_tags = port_state.get(LB.Port.VLAN_SUBTREE, {}).get(
-            LB.Port.Vlan.TRUNK_TAGS, []
-        )
+        port_vlan_state = port_state.get(LB.Port.VLAN_SUBTREE, {})
+        if not port_vlan_state:
+            return
+        port_type = port_vlan_state[LB.Port.Vlan.TYPE]
```

Comment:
vlan_type?

Reply:
I sincerely don't understand. Please elaborate.

Reply:
port_type should probably be vlan_type.

Reply:
ok, I had it as a 'trunk port' or 'access port', but I'd say yours fits the definition better. 

Let's move forward anyway :)